aws/compose.yml

```diff
@@ -0,0 +1,50 @@
+services:
+  orangehrm:
+    image: orangehrm/orangehrm:5.5
```

Comment:
```suggestion
    image: orangehrm/orangehrm
```
`latest`

Reply:
did not change this since version checking happens from here